recipes/r-rsvg/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = '1.1' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-rsvg
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: rsvg_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvg_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvg/rsvg_{{ version }}.tar.gz
+  sha256: 8ec457f1d52894619329650c58e2597b168e3312a1129d7863e23abad7d54e9e
+
+build:
+  number: 0
+  skip: true  # [win32]
```

Comment:
You'll have to make a judgement call here. Is this package worth installing on Windows if librsvg is not available? The librsvg-feedstock [skips Windows](https://github.com/conda-forge/librsvg-feedstock/blob/master/recipe/meta.yaml#L17) entirely.

Reply:
From the [Readme](https://github.com/jeroen/rsvg#readme) I figured out that librsvg is only needed in MacOS and Linux. We tested in Windows and did not run into an error when plotting the SVG from the example. I would also be fine with skipping Windows, though.

Reply:
If you tested it on Windows, then you should definitely keep the Windows build.

Reply:
Should be more precise here: We tested on x64, not x86 ;)